src/main/java/com/waa/project/entity/Event.java

```diff
@@ -9,23 +9,32 @@
 
 @Entity
 @Data
+@Table(name = "events")
 public class Event {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Column(name = "name", nullable = false)
     private String name;
+
+    @Column(name = "body", nullable = false)
     private String body;

```

Comment:
Can we please use the word `description` instead of `body` here as well?

Reply:
One more thing, is there a purpose of having these two fields separate?

   private LocalDate eventDate;
    private LocalTime eventTime;

I mean, can we combine them into one field?